koku/api/report/aws/openshift/provider_map.py

```diff
@@ -513,7 +304,7 @@ def __init__(self, provider, report_type):
                                 * Coalesce("exchange_rate", Value(1, output_field=DecimalField()))
                             ),
                             "count": Count("resource_id", distinct=True),
-                            "usage": Sum(F("usage_amount")),
+                            "usage": Sum("usage_amount"),
```

Comment:
Just curious what's the advantage of pulling the value into Python here.

Reply:
🤔 I don't honestly know how this change impacts things. But also, I'm going to put it back since this was not an intentional change.